README.md

```diff
@@ -4,3 +4,17 @@
 
 
 A Flask based web service used by the public to get access to Civil Legal Aid services.
+
+# MOJ Flask Skeleton
+
+The MOJ Flask Skeleton is set as a subtree from https://github.com/ministryofjustice/moj-flask-govuk-skeleton and is kept in the laa_access_civil_legal_aid folder.
+
+To retrieve the most recent updates from the flask skeleton, run:
+```shell
+git subtree pull --prefix=laa_access_civil_legal_aid https://github.com/ministryofjustice/moj-flask-govuk-skeleton.git main --squash
+```
+
+To contributing back to the MOJ Flask Skeleton Project with local updates:
```

Comment:
Should we put (not recommended) type explanation here or alternative way of doing this?

Reply:
I've taken squash out of the main command and added it as an optional extra so it doesn't automatically overwrite